applications/FluidDynamicsApplication/tests/couette_flow_test.py

```diff
@@ -134,8 +140,5 @@ def _AddReferenceValuesCheck(self):
 
 if __name__ == '__main__':
     test = CouetteFlowTest()
-    test.setUp()
     # test.testCouetteFlow2DSymbolicStokes()
```

Comment:
Do we have to deactivate the test in here? Can't we have them moved to nighly if required in the `test_FluidDynamicsApplication.py` file?

Reply:
mmm... I think I'm not getting your point... This is to run this test only without the need of running the suite (again I might be wrong).